### ISSUE TITLE
Locking/ re-setting

### ISSUE THREAD

**Issue by okdtoker** [leatrix/leatrix-plus-bcc#72]
Of my name icon and target, very annoying. Yes, I turned off all add-ons and enabled them one one by one until I started leatrix!

**Comment by leatrix** [comment]
Do you mean the player frame and target frame normally shown in the top-left corner?  Have you enabled 'Manage frames'?  If you enable 'Manage frames', you need to use 'Manage frames' to position the player frame and target frame.  Click the configuration next to 'Manage frames' and then you can drag and scale the player frame and target frame.

**Comment by okdtoker** [comment]
yes

**Comment by leatrix** [comment]
Disable all other addons.  Enable 'Manage frames'.  Click the reload button.  After the reload, click the configuration button for 'Manage frames' then position your frames.  Does it work?  If the configuration button is locked out, but 'Manage frames' is enabled, you need to click the reload button.

**Comment by okdtoker** [comment]
again? OK I will try it again

**Comment by okdtoker** [comment]
Still locked

**Comment by leatrix** [comment]
What's actually locked?

**Comment by leatrix** [comment]
If you're seeing 'Move frame', you're probably trying to use the default UI to move the player and target frame.  If you enable 'Manage frames', you cannot do that.

So either:
- Disable 'Manage frames' ('Frames') and reload.  Then you will be able to use the 'Move frame' setting in the player and target frame dropdown menus.
- Or, enable 'Manage frames' ('Frames') and reload.  Then you will be able to use the configuration button next to 'Manage frames' to drag and scale your frames.

It has to be one or the other.  If you enable 'Manage frames', you cannot use 'Move frame' because they would conflict.  You have to choose which method you want to use.  If you use 'Manage frames', you can scale the player and target frame and both frame positions will be saved account-wide.

**Comment by okdtoker** [comment]
Yes, once I figured out what you mean by Manage frames (thanks for not answering that BTW!) and to tell the truth, it is NOT up to you how I want to move ANYTHING, so I am turning off your Plus (annoyances) pak until you realize being obtuse and not answering questions is not a winning strategy
